src/app/components/livegrid.rs

```diff
@@ -3,8 +3,32 @@ use leptos::*;
 use leptos_router::*;
 use log::info;
 
+#[derive(Debug, Clone)]
 #[allow(dead_code)]
-type Items = Vec<String>;
+pub struct Item {
+  pub key: usize,
+  pub hash: ReadSignal<String>,
+  set_hash: WriteSignal<String>,
+}
+
+#[allow(dead_code)]
+impl Item {
+  pub fn new(cx: Scope, key: usize, hash: String) -> Self {
+    let (hash, set_hash) = create_signal(cx, hash);
```

Comment:
Tried `create_rw_signal` first and switched back to `create_signal` afterwards - both could not fix current issues we have for updating values in `For` component.